lib/download/index.js

```diff
@@ -1,18 +1,10 @@
+import fileDownload from 'js-file-download';
```

Comment:
The module we're delegating the file download responsibility. It has support for edge cases like Internet Explorer.